Frontend/src/components/Input/PopupFormular.svelte

```diff
@@ -46,6 +47,7 @@
                     on:change={(event) => {
                       if (input.onChange) input.onChange(event.detail);
                     }}
+                    required={(input.required ?? true) && !input.disabled}
```

Comment:
We should also implement this for AutocompleteInputs.

Reply:
I tried, but the library you included, doesn't support it (and sadly doesn't support any kind of validation too).

Reply:
Ah that's bad. But I think since we are not submitting forms we do not need the required html attribute on any inputs anyway. So there shouldn't be any change in behaviour if we just delete this required prop for the text field?

You can still validate / restrict the AutocompleteInput by referencing the input field directly like so: https://github.com/leih-lokal/LeihLokalVerwaltung/blob/master/Frontend/src/components/Input/AutocompleteInput.svelte#L16

Reply:
Hm? The whole point of the required attribute here is that empty fields will be colored differently as a hint for the user. It is valid for text fields, but yes we could still reference it directly.

Reply:
Hm, did you test that? For me it does not color the text fields. I think that is because we do not submit the form.